### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by theVivek-kumar**
single video page

### CODE REVIEW

File: src/context/Likedvideocontext.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  }
+
+  async function removeFromLikeVideo(videoId, setLikeVideo) {
+    try {
+      const response = await axios({
+        method: "DELETE",
+        url: `/api/user/likes/${videoId}`,
+        headers: { authorization: localStorage.getItem("token") },
+      });
+      if (response.status === 201) {
```
Comment (Adarsh-Patel-Dev) [inline src/context/Likedvideocontext.js]: here response status code should be 200

File: src/App.css [hunk 2]
```diff
@@ -55,23 +55,26 @@
...
- color: var(--PRIMERY-color);
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+gap: 3rem;
+margin-left: 7rem; 
+list-style: none;
+font-size: 3rem;
+color: var(--PRIMERY-color);
```
Comment (Adarsh-Patel-Dev) [inline src/App.css:66]: the color variable should be `(--PRIMARY-COLOR)`

File: src/App.css [hunk 3]
```diff
@@ -55,23 +55,26 @@
...
 }
+
 .icons-bar{
   font-size: 5rem;
+  color: var(--PRIMERY-color);
 }
 .main-hero-section{
     margin-top: .2rem;
```
Comment (Adarsh-Patel-Dev) [inline src/App.css:74]: For less than 1 rem use px because you don't need to scale any element whose size < 1rem

File: src/App.css [hunk 4]
```diff
@@ -219,3 +236,56 @@
...
+display: grid;
+grid-template-columns: 17rem 1fr;
+}
+.duration-label {
+    position: relative;
+    background-color: #222222d8;
+    color: white;
+    padding: 3px 5px;
+    border-radius: 5px;
+    bottom: 30px;
```
Comment (Adarsh-Patel-Dev) [inline src/App.css:260]: Prefer rem where element size >10px

File: src/App.js [hunk 5]
```diff
@@ -1,14 +1,23 @@
 import "./App.css";
+// import MockAPI from './mockman/MockAPI'
+import { Routes, Route } from "react-router-dom";
 import "./components/navbar.css"
 import LandingPage from "./page/landingPage";
 import SingleVideoPage from "./page/SingleVideoPage";
 import VideoListing from "./page/videoListing";
+import WatchLaterpage from "./page/WatchLaterpage";
+import { HistoryPage } from "./page/History-page";
```
Comment (Adarsh-Patel-Dev) [inline src/App.js:9]: Don't mix named or default import/import in your projects . 
Either use named or default export/import.

File: src/components/MainArea.js [hunk 6]
```diff
@@ -1,10 +1,77 @@
...
+import { MdPlaylistPlay ,MdOutlineWatchLater} from 'react-icons/md';
+import Card from './Videocard';
+
 
 function MainArea() {
+  
+  const { videoId } = useParams();
+  const { videoList } = useVideoListing();
+  const filteredVideo = videoList.filter((video) => video._id === videoId);
+  // const {video} = undefined || {};
```
Comment (Adarsh-Patel-Dev) [inline src/components/MainArea.js:13]: remove this comment

**Review by Adarsh-Patel-Dev — APPROVED** [review]
Suggested some changes in the code. 
Live link is missing , you can add the screenshots of UI for reference
You need to format the code. Apart from that whole code is good to merge

File: src/page/videoListing.jsx [hunk 7]
```diff
@@ -1,15 +1,61 @@
...
+      <div className='main-section-wrapper'>
+        <SideMenu />
+              <div class="hero-section-wrapper">
+              
+                    
+                
+                        <Chip 
+                           
+                            />
+
```
Comment (swethaogeti) [inline src/page/videoListing.jsx:26]: remove extra spaces

**Review by swethaogeti — COMMENTED** [review]
remove unused imports , check the indentation and rest looks clean.

**Pull request merged**